purchase_order_secondary_unit/views/purchase_order_views.xml

```diff
@@ -15,16 +15,19 @@
                     <field
                         name="secondary_uom_qty"
                         class="oe_inline oe_no_button"
-                        attrs="{'readonly': [('state', 'in', ('done', 'cancel'))]}"
+                        groups="uom.group_uom"
+                        readonly="state in ['done', 'cancel']"
                     />
                     <field
                         name="secondary_uom_id"
                         class="oe_inline"
-                        domain="product_id and ['|', ('product_id', '=', product_id),
+                        domain="['|', ('product_id', '=', product_id),
```

Comment:
`domain` definition is still the old one, changes added by https://github.com/OCA/purchase-workflow/commit/ee8f610f2fed6f4456a5b4c8c131883e511170ac are lost, please check diff at this point.